dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelAIPluginExtensions.cs

```diff
@@ -348,7 +355,14 @@ async Task<SKContext> ExecuteAsync(SKContext context)
                     }
                 }
 
-                var result = await runner.RunAsync(operation, arguments, cancellationToken).ConfigureAwait(false);
+                var configuration = new RestApiOperationRunConfig(arguments)
+                {
+                    ServerUrlOverride = executionParameters?.ServerUrlOverride,
+                    DocumentUri = documentUri
```

Comment:
Done, although I left `DocumentUri` on higher levels since on that stage it's not just host URL, but full URI where OpenAPI document exists. So, when I initialize `RestApiOperationRunOptions`, I'm converting document URI to host URL by using `GetLeftPart` method.